lookup.go

```diff
@@ -1,14 +1,18 @@
 package messagebird
 
-import "net/url"
+import (
+	"net/url"
```

Comment:
If there is a single import there is no need for import block:
`import "net/url"`